partially remove name spaces

edewit opened the pull request:
this removes the prefixes for

* client-scopes
* client-scopes-help
* clients-help
* clients

working towards fixing #23094

jonkoops reviewed (changes requested):
The messages for locales other than English have not been updated. It also seems like there are a lot of unrelated formatting changes (perhaps a `pnpm i` is needed?).

edewit committed:
this removes the prefixes for

client-scopes
client-scopes-help
clients-help
clients
working towards fixing #23094
